librz/arch/p/asm/asm_mips.c

```diff
@@ -3,32 +3,28 @@
 
 #include <rz_asm.h>
 #include <rz_lib.h>
-#include <mips/mips_assembler.h>
+#include <mips/mips_internal.h>
+#include "capstone.h"
```

Comment:
```suggestion
#include <capstone/capstone.h>
```

Reply:
that was added by the IDE, i have to change it

Reply:
Yeah, happens to me all the time as well. Annoying.